prebuilt-tasks/src/main/java/com/redhat/parodos/tasks/tibco/Tibjms.java

```diff
@@ -0,0 +1,8 @@
+package com.redhat.parodos.tasks.tibco;
+
+public interface Tibjms {
```

Comment:
nit: TibJMS ?

Reply:
comeon...tib is also an abbreviation. This is how they do it. LEt's keep it